editor/components/post-title/style.scss

```diff
@@ -35,8 +35,9 @@
 
 .editor-post-title .editor-post-permalink {
 	font-size: $default-font-size;
+	color: $dark-gray-900;
 	position: absolute;
-	top: -35px;
+	top: -$block-toolbar-height + 1px + 1px; // move upwards the toolbar height, but subtract borders
```

Comment:
This comment is a bit awkwardly-worded; how about:

```scss
top: -$block-toolbar-height + 1px + 1px; // Shift this element upward the same height as the block toolbar, minus the border size
```

All that said, add the `1px` in there twice is weird. Why not just make a `$border-width` variable equal to `1px`, then do `-$block-toolbar-height + ($border-width * 2)`? It'd be easier to read.